src/dev/flang/be/interpreter/Intrinsics.java

```diff
@@ -438,8 +438,28 @@ private static synchronized void releaseFileDescriptor(long fileDescriptor)
               stats[3] = metadata.isDirectory()? 1:0;
               return new boolValue(true);
             }
-          catch (Exception e)
+          catch (UnsupportedOperationException e)
+            {
+              stats[0] = 95; // ENOTSUP 95 Operation not supported
```

Comment:
Can you add a class or enum that defines named constants for errno? As a general rule, any number except 0 and maybe 1 should usually be declared as a named constant.